tests/github/test_hello_federation.sh

```diff
@@ -6,7 +6,7 @@ FED_WORKSPACE=${2:-'fed_work12345alpha81671'}   # This can be whatever unique di
 COL1=${3:-'one123dragons'}  # This can be any unique label (lowercase)
 COL2=${4:-'beta34unicorns'} # This can be any unique label (lowercase)
 
-FQDN=${5:-$(hostname --all-fqdns | awk '{print $1}')}
+FQDN=${5:-$(hostname --fqdn | awk '{print $1}')}
```

Comment:
This line should be reverted

Reply:
Done.

Although on my machine the --fqdn correctly got the FQDN whereas --all-fqdns returned nothing. Maybe if the FQDN is an empty string, then it can try --fqdn as a backup?